src/test/integration/tests/returnvalues.js

```diff
@@ -70,11 +70,6 @@ module.exports = async function(ctx) {
   toggleScopes(dbg);
   await togglePauseOnExceptions(dbg, true, false);
 
-  const TESTS = ["57", "0", "false", "undefined", "null", "\"undefined\""];
```

Comment:
This change loses the tests for 0 and null, but I think those are important to keep as they are falsy values, and so easy to get wrong.  Probably we should even expand this to add the empty string, which I forgot about in the earlier patch.